cmd/create/cluster/cmd.go

```diff
@@ -266,10 +265,10 @@ func run(cmd *cobra.Command, _ []string) {
 	// Validate AWS credentials for current user
 	reporter.Infof("Validating AWS credentials for CFUser...")
 	if err = client.ValidateCFUserCredentials(); err != nil {
-		reporter.Errorf("Error validating AWS credentials: %v", err)
+		reporter.Errorf("Error validating OSDCcsAdmin user AWS credentials: %v", err)
```

Comment:
The name is `osdCcsAdmin`, but really you should use the `aws.AdminUserName` const here:
```go
reporter.Errorf("Error validating AWS credentials for user '%s': %v", aws.AdminUserName, err)
```